services/auth/src/static-scopes.json

```diff
@@ -2,44 +2,40 @@
   {
     "clientId": "static/taskcluster/built-in-workers",
     "scopes": [
-      "queue:claim-work:built-in/*",
       "assume:worker-id:built-in/*",
-      "queue:worker-id:built-in/*",
-      "queue:resolve-task"
+      "queue:claim-work:built-in/*",
+      "queue:resolve-task",
+      "queue:worker-id:built-in/*"
     ]
   },
   {
     "clientId": "static/taskcluster/github",
     "scopes": [
       "assume:repo:github.com/*",
-      "queue:scheduler-id:taskcluster-github",
-      "queue:route:statuses",
-      "queue:route:checks",
       "assume:scheduler-id:taskcluster-github/*",
-      "auth:azure-table-access:${azureAccountId}/TaskclusterGithubBuilds",
-      "auth:azure-table-access:${azureAccountId}/TaskclusterIntegrationOwners",
+      "auth:azure-table:read-write:${azureAccountId}/TaskclusterCheckRuns",
+      "auth:azure-table:read-write:${azureAccountId}/TaskclusterChecksToTasks",
       "auth:azure-table:read-write:${azureAccountId}/TaskclusterGithubBuilds",
```

Comment:
how was this even working before?

Reply:
the former scope implies the latter, it's just the legacy form.